manifests/monitoring/kruize-monitoring-openshift.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: autotune
+  labels:
+    app: autotune
+  namespace: openshift-tuning
+spec:
+  replicas: 1
+  selector:
+      matchLabels:
+        name: autotune
+  template:
+    metadata:
+      labels:
+        app: autotune
+        name: autotune
+        operatorframework.io/os.linux: supported
+    spec:
+      containers:
+      - name: autotune
+        image: "quay.io/khansaad/autotune_operator:0.0.8_mvp"  # will be updated later
+        imagePullPolicy: Always
+        env:
+          - name: CLUSTER_TYPE
+            value: "kubernetes"
+          - name: K8S_TYPE
+            value: "openshift"
+          - name: AUTH_TYPE
+            value: ""
+          - name: AUTH_TOKEN
+            value: ""
+          - name: MONITORING_AGENT
+            value: "prometheus"
+          - name: MONITORING_SERVICE
+            value: "prometheus-k8s"
+          - name: MONITORING_AGENT_ENDPOINT
+            value: "prometheus-k8s"
+          - name: LOGGING_LEVEL
+            value: "info"
+          - name: ROOT_LOGGING_LEVEL
+            value: "error"
```

Comment:
Should this be error or info?

Reply:
Initially we want INFO messages to be turned on a well as this will aid in debugging

Reply:
Changed to INFO